src/views/RegisterPage/index.js

```diff
@@ -106,68 +107,70 @@ class RegisterPage extends Component {
   render() {
     return (
       <LoginRegisterLayout type="register" title="Register">
-        <InputField
-          isRequired={true}
-          type={"text"}
-          name={"firstname"}
-          placeholder={"First Name"}
-          value={this.state.firstname}
-          handleChange={this.handleChange}
-        />
-        <InputField
-          isRequired={true}
-          type={"text"}
-          name={"lastname"}
-          placeholder={"Last Name"}
-          value={this.state.lastname}
-          handleChange={this.handleChange}
-        />
-        <InputField
-          isRequired={true}
-          type={"email"}
-          name={"email"}
-          placeholder={"Email"}
-          value={this.state.email}
-          handleChange={this.handleChange}
-        />
-        <InputField
-          isRequired={true}
-          type={"password"}
-          name={"password"}
-          placeholder={"Password"}
-          value={this.state.password}
-          handleChange={this.handleChange}
-        />
-        <InputField
-          isRequired={true}
-          type={"password"}
-          name={"confirmpassword"}
-          placeholder={"Confirm Password"}
-          handleChange={this.handleChange}
-        />
+        <Form style={{ width: "100%" }}>
```

Comment:
I think just need to add the "onKeyDown" part here too (like in LoginPage) so it responds to enter

Reply:
Yup, good catch

Reply:
Oh actually you know what it is - it's the `event.preventDefault();` in the `handleSubmit(event)` function. Looks like enter triggers handleSubmit, and the default behaviour is to refresh the page. No need for `onKeyDown` at all.